evals/cybench/agent_execution_environment/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# This defines a docker image which the agent can run commands in.
```

Comment:
Maybe name this directory `agent` rather than `agent_execution_environment` (I think it will disambiguate well enough with that name and it make the top level filenames more aesthetically consistent.

Reply:
I'm happy to do this, but I also slightly worry that it may be misleading in that the actual logic for the agent lives in `task.py` (or wherever users define their `Plan` / list of `Solver`s).

Reply:
That's true! What other pithy term could we use? Maybe `attacker` or `client`?

Reply:
To me `agent_environment` seems most precise, but I'm not too opinionated about this and would be happy to go with one of those two names or something else.

Reply:
We used `attacker` in the GDM CTFs. I like it because it's clear and pithy (`agent_environment` is a bit too abstract / RL sounding)

Reply:
I'm not familiar with what term the general industry/literature tends to use, but I'd lean towards whatever that might be if there is a clear favourite. I think `agent_execution_environment` used to make sense when we referred to these as execution environments (before tool environments and before sandbox environments). 

If you're concerned about `agent` being mistaken for a Python package, perhaps `agent_sandbox`? I like `attacker` too personally.

Reply:
We've used attacker to refer to other environments which host attacking servers (like a server hosting a CSRF attack) so it could become a bit overloaded

Reply:
I do think `agent_environment` or `agent_sandbox` both describe the thing pretty accurately. Also `attacker` presupposes the set up of the eval, we have a bunch of investigative evals which don't attack a machine and may in the future design defensive evals using this template